lib/ui/views/projects/project_details_view.dart

```diff
@@ -64,6 +66,20 @@ class _ProjectDetailsViewState extends State<ProjectDetailsView> {
     );
   }
 
+  Widget _buildOpenActionButton() {
+    return Padding(
+      padding: const EdgeInsets.symmetric(horizontal: 8),
+      child: IconButton(
+        onPressed: () async {
+          final url = Uri.parse(
+              'https://circuitverse.org/users/${widget.project.relationships.author.data.id}/projects/${widget.project.id}');
+          await launchUrl(url, mode: LaunchMode.externalApplication);
```

Comment:
There is already an predefined function for launchUrl in lib/utils/url_launcher.dart, kindly use that function to open the link.

Reply:
i have changed the code accordingly , please review the code.